misc/bazel/rust.bzl

```diff
@@ -0,0 +1,23 @@
+load("@rules_rust//rust:defs.bzl", "rust_binary")
+load("@semmle_code//buildutils-internal:glibc_symbols_check.bzl", "glibc_symbols_check")
+load("@semmle_code//buildutils-internal:lipo.bzl", "universal_binary")
+
+def codeql_rust_binary(
+        name,
+        target_compatible_with = None,
+        visibility = None,
+        symbols_test = True,
+        **kwargs):
+    rust_label_name = name + "_single_arch"
```

Comment:
wouldn't it be maybe better to use `single_arch/name`, to get a consistent binary base name?